backup-codelab-done/app/src/main/res/xml/backup_descriptor.xml

```diff
@@ -1,19 +1,4 @@
 <?xml version="1.0" encoding="utf-8"?>
-<!--
```

Comment:
Why remove the license?

Reply:
I generated from scratch and didn't add it back :).

Reply:
fixed